packages/open-lens/webpack/dev-server.ts

```diff
@@ -23,20 +23,25 @@ const server = new WebpackDevServer({
   allowedHosts: ".lens.app",
   host: "localhost",
   port: webpackDevServerPort,
-  static: buildDir, // aka `devServer.contentBase` in webpack@4
-  hot: "only", // use HMR only without errors
+  static: {
+    directory: buildDir,
+    serveIndex: true,
+  },
+  hot: true,
   liveReload: false,
+  // historyApiFallback: true,
   devMiddleware: {
     writeToDisk: true,
     index: "index.html",
     publicPath: "/build",
   },
-  proxy: {
-    "^/$": "/build/",
-  },
+  // proxy: {
+  //   "^/$": "/build/",
+  // },
```

Comment:
@jakolehm why do we need this `proxy` at all? any ideas? it seem like works fine without this..